tests/logictest/suites/gen/02_function/02_0001_function_type_of

```diff
@@ -124,19 +124,19 @@ select '=== TEST_datetimes';
 ----  
 === TEST_datetimes
 
-statement query B label(mysql,http)
+statement query B label(http)
```

Comment:
does this case run for clickhouse?

Reply:
> does this case run for clickhouse?

Yes, without query skipped or skipif clickhouse, the case will run on all handlers.

Reply:
clickhouse handler use the same expect results with http handler

Reply:
> clickhouse handler use the same expect results with http handler?

Results without any labels is default expect.